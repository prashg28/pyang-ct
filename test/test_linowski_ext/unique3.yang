 module unique {

   namespace "urn:org:example:xml:ct:unique";
   prefix "u";

   import complex-types {prefix "ct"; }

   ct:complex-type  c1 {
       leaf n1 {
           type int8;
       }
       leaf n1 {
           type int8;
       }
   }

   ct:instance c {
       ct:instance-type c1;
   }


}
