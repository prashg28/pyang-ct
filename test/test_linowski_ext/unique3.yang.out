unique3.yang:12: error: DUPLICATE_CHILD_NAME
